plone-6.0.x.cfg

```diff
@@ -1,11 +1,8 @@
 [buildout]
 extends =
-    https://dist.plone.org/release/6.0.0a2/versions.cfg
+    https://dist.plone.org/release/6.0.0a4/versions.cfg
     base.cfg
 
-find-links += https://dist.plone.org/release/6.0.0a2/
-
-
 [instance]
 recipe = plone.recipe.zope2instance
 zodb-temporary-storage = off
```

Comment:
This tempstorage line can be removed now, improved in Plone.recipe.zope2instance.